### PR TITLE
Revert changes to numeric-testcommon.js from f00b5b0 

### PR DESCRIPTION
**Pull request by nt1m**
They regressed Safari Interop 2023 results (which is a Safari bug, but out of scope for this year).

### CODE REVIEW

**Review by emilio — CHANGES_REQUESTED** [review]
If you want to keep using transform, you also need to fix the extractValue implementations. Curious, what's the WebKit bug?

File: css/support/numeric-testcommon.js [hunk 1]
```diff
@@ -122,6 +130,10 @@ function test_math_specified(testString, expectedString, {approx, msg, msgExtra,
         assert_not_equals(expectedValue, '', `${expectedString} isn't valid in '${prop}'; got the default value instead.`)
         if (approx) {
             let extractValue = function(value) {
+                let escapeRegex = (string) => {
+                    return string.replace(/[/\-\\^$*+?.()|[\]{}]/g, "\\$&");
```
Comment (emilio) [inline css/support/numeric-testcommon.js:134]: This probably needs a comment... But is this correct? The previous code would deal with the used `transform` values (so, `matrix(...)`, while this code seems like it wouldn't).

**Review by emilio — CHANGES_REQUESTED** [review]

**Comment by emilio** [comment]
It seems the safari bug is that translate/rotate/etc don't accept math functions or so? If so it might be easier to fix than this tbh. Might even give it a shot this weekend if I find the time :)

**Comment by nt1m** [comment]
@emilio I looked more into detail into why these tests are failing in Safari. It's just a serialization issue. The computed value of `scale: 1` is `scale: none`. Which arguably isn't incorrect, but trips over all the tests.

**Comment by nt1m** [comment]
Can we workaround this issue still? Potentially by finding another property to use instead in order to test numbers (like line-height ?)

**Comment by emilio** [comment]
> @emilio I looked more into detail into why these tests are failing in Safari. It's just a serialization issue. The computed value of `scale: 1` is `scale: none`. Which arguably isn't incorrect, but trips over all the tests.

It is completely incorrect. scale: none doesn't create a containing block for fixed pos descendants etc

**Comment by emilio** [comment]
> Can we workaround this issue still? Potentially by finding another property to use instead in order to test numbers (like line-height ?)

Not easily because line heights return pixels from gCS. I looked at opacity but it doesn't allow negatives so that's also not good enough. But sure if you find another property that takes and computes to numbers, that sounds good.

**Comment by nt1m** [comment]
I'm not going to pursue this, it's no longer needed to defend either Safari's experimental or stable score.

**Pull request closed without merging**